azion/stage/azion.json

```
@@ -142,6 +147,11 @@
         "id": 263041,
         "name": "Secure Headers",
         "phase": "response"
+      },
+      {
+        "id": 267300,
+        "name": "Route GraphQL Billing Queries to Manager Origin",
+        "phase": ""
```

Empty phase? This should be
```suggestion
        "phase": "request"
```
right?